packages/ses/docs/lockdown.md

```diff
@@ -26,7 +26,8 @@ Each option is explained in its own section below.
 | `localeTaming`   | `'safe'`         | `'unsafe'`     | `toLocaleString`           |
 | `consoleTaming`  | `'safe'`         | `'unsafe'`     | deep stacks                |
 | `errorTaming`    | `'safe'`         | `'unsafe'`     | `errorInstance.stack`      |
-| `errorTrapping`  | `'platform'`     | `'exit'` `'abort'` `'report'` | handling of uncaught exceptions |
+| `errorTrapping`  | `'platform'`     | `'exit'` `'abort'` `'report'` `'none'` | handling of uncaught exceptions |
+| `unhandledRejectionTrapping`  | `'none'`     | `'exit'` `'abort'` `'report'` | handling of finalized unhandled rejections |
```

Comment:
In the table, this differs from the `errorTrapping` option by omitting `platform`. But the documentation of this option below still mentions platform.

Having this default to `'none'` seems wrong. Why not at least `'report'`? (Below I also ask whether the actual platform behavior should be an option.)

Reply:
Changed to default to `'report'`.